script/travis_run_js_tests.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+set -ev
+
+if [[ $( git diff --name-only origin/master..HEAD webpack/ .travis.yml .babelrc .eslintrc package.json | wc -l ) -ne 0 ]]; then
```

Comment:
It should not be against the master branch

Reply:
good catch, fixed!